addons/web/static/src/js/view_list.js

```diff
@@ -2305,13 +2305,9 @@ instance.web.list.Binary = instance.web.list.Column.extend({
         var text = _t("Download");
         var value = row_data[this.id].value;
         var download_url;
-        if (value && value.substr(0, 10).indexOf(' ') == -1) {
-            download_url = "data:application/octet-stream;base64," + value;
```

Comment:
IMO the cons of the data url (=no download filename => no filetype => downloaded as raw binary data) outweighs the pro (not downloading the file again). How many times does the user click on the same download link multiple times?

If the download attribute is implemented and it works, perfect. Until then, I (and I think most users) would greatly prefer a working download link that redownloads the entire file if I happen to click on it twice instead of a broken download that doesn't.